screen/edit-transaction/src/main/java/com/ivy/transaction/EditTransactionViewModel.kt

```diff
@@ -595,7 +598,7 @@ class EditTransactionViewModel @Inject constructor(
         viewModelScope.launch {
             ioThread {
                 loadedTransaction?.let {
-                    transactionWriter.flagDeleted(it.id)
+                    transactionRepo.flagDeleted(TransactionId(it.id))
```

Comment:
In a follow-up task, for the sake of simplicity we can replace transaction flagging with direct deletion instead

Reply:
Can you tell me how can we process ahead on this follow-up , or like immediate measures or is it something requires to work upon for this particular github issue or we can create another one for this etc..

Reply:
Yes, correct